src/applications/personalization/profile/util/direct-deposit.js

```diff
@@ -14,44 +15,42 @@ export class DirectDepositClient {
     return this.useLighthouseEndpoint ? this.#LH_ENDPOINT : this.#PPIU_ENDPOINT;
   }
 
-  generateApiRequestOptions(payload) {
-    set(payload, 'financialInstitutionName', 'Hidden form field');
+  generateApiRequestOptions(fields) {
+    // The PPIU endpoint REQUIRES a financialInstitutionName field, but the
+    // Lighthouse endpoint does not. We set a dummy value here to avoid
+    // validation errors aka 500s from vets-api
+    set(fields, 'financialInstitutionName', 'Hidden form field');
 
     const options = {
       headers: {
         'Content-Type': 'application/json',
       },
       method: 'PUT',
-      body: JSON.stringify(payload),
+      body: JSON.stringify(fields),
       mode: 'cors',
     };
 
     if (this.useLighthouseEndpoint) {
       options.body = JSON.stringify({
-        accountType: payload.accountType,
-        accountNumber: payload.accountNumber,
-        routingNumber: payload.financialInstitutionRoutingNumber,
+        accountType: fields.accountType,
+        accountNumber: fields.accountNumber,
+        routingNumber: fields.financialInstitutionRoutingNumber,
       });
     }
     return options;
   }
 
   formatDirectDepositResponseFromLighthouse = response => {
     const result = cloneDeep(response);
-    if (result?.paymentAccount?.name) {
-      set(
-        result,
-        'paymentAccount.financialInstitutionName',
-        result.paymentAccount.name,
-      );
-    }
-    if (result.paymentAccount?.routingNumber) {
-      set(
-        result,
-        'paymentAccount.financialInstitutionRoutingNumber',
-        result.paymentAccount.routingNumber,
-      );
-    }
+    set(result, 'paymentAccount', {
+      financialInstitutionName: result?.paymentAccount?.name,
+      financialInstitutionRoutingNumber:
+        response?.paymentAccount?.routingNumber,
+      accountNumber: response?.paymentAccount?.accountNumber,
+      accountType:
+        capitalize(response?.paymentAccount?.accountType) || undefined,
```

Comment:
capitalizing is needed because lighthouse is returning the accountType as all CAPS, which isn't ideal for UI display.